resources/js/components/UserEditForm.vue

```diff
@@ -357,13 +376,23 @@ export default {
         });
     },
   },
+  computed: {
+    graduationInfoRequired: function() {
+      if (this.user.primary_affiliation == "Student") {
+        return false;
+      } else {
+        return true;
+      }
```

Comment:
```suggestion
      return this.user.primary_affiliation === "student";
```

Reply:
wouldn't it be
`return !this.user.primary_affiliation === "student";`